jquery/exercise4/js/inputHint.js

```diff
@@ -0,0 +1,43 @@
+function InputHint(mainIdDiv) {
+  this.$mainIdDiv = mainIdDiv;
+  this.$label = mainIdDiv.find("label");
+  this.labelText = this.$label.text();
+}
+
+InputHint.prototype.removeLabel = function() {
+  /*Set the value of the search input to the text of the label element*/
+  this.$label.next().val(this.labelText);
+  /*Add a class of "hint" to the search input*/
+  this.$label.next().attr("class", "hint");
+  /*Remove the label element*/
+  this.$label.remove();
+};
+
+InputHint.prototype.bindEvents = function() {
+  this.removeLabel();
```

Comment:
not the job of bindEvents function to do anything other than binding events. It should go to initialize section